connector/rest/src/test/java/it/pagopa/selfcare/dashboard/connector/rest/MsCoreConnectorImplTest.java

```diff
@@ -910,4 +916,83 @@ void findInstitutionsByProductIdAndType() {
 
     }
 
+    @Test
+    void getDelegationUsingFrom_shouldGetData() {
+        // given
+        DelegationResponse delegationResponse = dummyDelegationResponse();
+        List<DelegationResponse> delegationResponseList = new ArrayList<>();
+        delegationResponseList.add(delegationResponse);
+        ResponseEntity<List<DelegationResponse>> delegationResponseEntity = new ResponseEntity<>(delegationResponseList, null, HttpStatus.OK);
+        Delegation delegation = dummyDelegation();
+
+        when(msCoreInstitutionApiRestClient._getDelegationsUsingGET(any(), any(), any()))
+                .thenReturn(delegationResponseEntity);
+
+
+        // when
+        List<Delegation> delegationList = msCoreConnector.getDelegations(delegation.getFrom(), delegation.getTo(), delegation.getProductId());
+        // then
+        assertNotNull(delegationList);
+        assertEquals(1, delegationList.size());
+
+        assertEquals(delegationResponseList.get(0).getId(), delegationList.get(0).getId());
+        assertEquals(delegationResponseList.get(0).getFrom(), delegationList.get(0).getFrom());
+        assertEquals(delegationResponseList.get(0).getTo(), delegationList.get(0).getTo());
+        assertEquals(delegationResponseList.get(0).getProductId(), delegationList.get(0).getProductId());
+        assertEquals(delegationResponseList.get(0).getType().toString(), delegationList.get(0).getType().toString());
+        assertEquals(delegationResponseList.get(0).getInstitutionFromName(), delegationList.get(0).getInstitutionFromName());
+        assertEquals(delegationResponseList.get(0).getInstitutionToName(), delegationList.get(0).getInstitutionToName());
+
+        verify(msCoreInstitutionApiRestClient, times(1))
+                ._getDelegationsUsingGET(any(), any(), any());
```

Comment:
dont use any on the verify, unless impossible to do otherwise, always better to check if the passed parameters are the one you expect to be passed.